docs/README.md

```diff
@@ -1,11 +1,15 @@
 # Introduction
 
-math.gl is a JavaScript class library for 3D math. It provides the traditional 3D classes (vectors, matrices etc).
+math.gl is a JavaScript class library for 3D and geospatial math. It provides a set of "traditional" 3D classes for vectors, matrices etc, as well as a set of optional modules that address different domains.
 
 In spite of the name, math.gl has no hard WebGL dependencies and is usable in any JavaScript applications. That said, math.gl is designed to be optimized for WebGL/3D applications.
 
+## Available Modules
 
-## Features
+- `math.gl` - basic math classes (vectors, matrices etc).
```

Comment:
why does this not follow our other lib's convention, `@math.gl/core`?

Reply:
> why does this not follow our other lib's convention, @math.gl/core?

Backwards compatibility is the main reason. Also I am trying to make the additional modules usable without the "core" so it is not really a core.